Code/GraphMol/StructChecker/StructCheckerOptions.cpp

```diff
@@ -351,7 +351,8 @@ static bool ReadAAPairs(
     return false;
   }
   unsigned n = 0;
-  fscanf(fp, "%d", &n);
+  int num_scan = fscanf(fp, "%d", &n);
```

Comment:
I'm surprised this is needed, but we should probably assert num_scan == 1 at least, otherwise I expect the file is pretty broken.

Reply:
Yeah, it also surprised me that this caused a warning.

Reply:
I think the issue is that you aren’t using the fscanf return.  If the function wasn’t fscanf, I bet you wouldn’t get this error.  In any case, why not check that num_scan = 1 and fix a bug while your at it :)

Reply:
I'll add the `assert`.